src/convertion/safari-rules.ts

```diff
@@ -59,42 +64,54 @@ export function convertFilter(filter: NetworkFilter) {
   }
 
   const trigger = {};
+  const punycode = require('punycode');
```

Comment:
Not need to import this several time, you can import it from the top of this file.